local/nginx/include/rates.conf

```diff
@@ -1,3 +1,5 @@
 
 limit_req_zone $uri zone=tier_b:10m rate=40r/m;
 limit_req_zone $uri zone=tier_c:10m rate=5r/m;
+
+limit_req_zone $binary_remote_addr zone=get_entities:10m rate=30r/m;
```

Comment:
Is the rate limit using the x-forwarded-address or the IP address of the connection directly?

Reply:
Hmm I think `$http_x_forwarded_for` should be used instead of `$binary_remote_addr` for that purpose. To make cloudflare work. The real addresses hitting the catalysts are only this few ranges: https://www.cloudflare.com/ips/

Reply:
Oops, reading it from `X-Forwarded-For` header would be okay?

Reply:
Good

Reply:
yes

Reply:
That would not work for catalysts not using Cloudflare... and for those cases, adding a header in CURL will suffice to skip the rate limit